src/controllers/document.download.controller.ts

```diff
@@ -35,7 +36,7 @@ export const get = async (req: Request, res: Response) => {
     return download.data.pipe(res);
 
   } catch (e) {
-    logger.errorRequest(req, `download.attachment.controller - ${JSON.stringify(e)}`);
+    logger.errorRequest(req, `download.attachment.controller - ${inspect(e)}`);
```

Comment:
'e' contained circular entries and Stringify cannot handle that so was falling over.
Inspect replaces any circular entries with [circular]

Reply:
A one line comment in the code (immediately above) explaining this would be useful I think.